### DISCUSSION TITLE
Mapping one virtual button to toggle between two different Reaper actions?

### DISCUSSION THREAD

**Discussion by vonglan** [helgoboss/helgobox#567] [Q&A]
Is this possible?

Use case: X-Touch button to switch between track recording modes "MIDI overdub" and "MIDI replace".
Restriction: mappings should be in Main Mapping, based on virtual button as source.

**Comment by tompad2** [comment]
Using Cycle Editor Actions in Reaper?

Or using SWS/NF: Cycle through MIDI recording modes?
